localsettings.example.py

```diff
@@ -215,7 +215,7 @@
 
 # on both a local and a distributed environment this should be localhost
 ELASTICSEARCH_HOST = 'localhost'
-ELASTICSEARCH_PORT = 9200
+ELASTICSEARCH_PORT = 6200       # 6200 for ES2, 9200 for ES7
```

Comment:
I think this is true iff you are using the docker container, otherwise it's not. Maybe this needs a line in the docs for people not using docker.

Reply:
This setting is deleted in https://github.com/dimagi/commcare-hq/pull/29626. Maybe we can either make this change in `dev_settings` or we can get to some consensus about what the right default ES version setup is for dev environments, and make sure our docker and settings support that out of the box

Reply:
Heh, I saw that PR and I thought I made this change in `settings.py` - hazards of making PRs in the web ui with a lot of tabs open. But anyway, you're right that dev_settings is better. Based on slack it sounds like you're updating a few ES-related settings. It this PR still useful (assuming I move this to dev settings)?

Reply:
I imagine if there's no time sensitivity we can drop this in favor of the larger update that was discussed on slack (which I think would obsolete the change).

Reply:
👍